bigtable/quickstart/README.md

```diff
@@ -0,0 +1,78 @@
+# Cloud Bigtable Quickstart in Go
+
+This is a simple application that demonstrates using the [Google Cloud APIs Go
```

Comment:
"This application demonstrates..."

I've been trying to avoid words like "basic," and "simple," recently.

Reply:
done.